discovery-provider/src/queries/get_top_users.py

```diff
@@ -1,23 +1,32 @@
-from sqlalchemy.sql.expression import desc
-from src.models.models import AggregateUser, User
-from src.queries.query_helpers import helpers, paginate_query, populate_user_metadata
+from src.queries.query_helpers import get_pagination_vars, populate_user_metadata
 from src.utils.db_session import get_db_read_replica
 
+sql = """
```

Comment:
Seems like a case of sql alchemy kinda shooting us in the foot. +1 to the raw sql

Reply:
wait there's a case where sqlalchemy doesn't shoot you in the foot? 🤣